config/routes.rb

```diff
@@ -561,7 +565,21 @@
         # API
         namespace :api, path: "/" do
           namespace :v1, constraints: ::ApiConstraint.new(version: 1, default: true), path: "/api/v1/data" do
-            get "/" => "query#index", as: :root
+            get "/" => "query#index", as: :root, defaults: { format: "json" }
```

Comment:
This route, according to the docs should be `
GET /api/v1/data/data{.json|.csv)`